storey/writers.py

```diff
@@ -19,8 +20,8 @@
 
 class _Writer:
     def __init__(self, columns: Union[str, List[str], None], infer_columns_from_data: Optional[bool],
-                 index_cols: Union[str, List[str], None] = None, retain_dict: bool = False,
-                 storage_options: Optional[dict] = None):
+                 index_cols: Union[str, List[str], None] = None, partition_cols: Union[str, List[str], None] = None,
+                 retain_dict: bool = False, storage_options: Optional[dict] = None):
```

Comment:
please add documentation for these new params

Reply:
There's only one new param :)

Reply:
And it's actually already documented 😄 
Just need to update the documentation.

Reply:
Updated.